lua/orgmode/objects/priority_state.lua

```diff
@@ -29,13 +29,18 @@ function PriorityState:prompt_user()
   local choice = vim.fn.input(prompt)
 
   if choice == '' then
+    utils.echo_warning(string.format("Priority must be between '%s' and '%s'", self.high_priority, self.low_priority))
```

Comment:
If the user just pressed `<Enter>`, most likely they wanted to cancel the priority addition.
Maybe don't warn about it? What do you think?

Reply:
Emacs prints this string when you `<Enter>` without any input, so I think it's better to stick with that. However, one could argue that this isn't really worth a warning and more an info. Emacs also adds no color indicating a warning, but I'm not sure if it ever does for warnings.

Reply:
OK